checks.d/system_swap.py

```diff
@@ -0,0 +1,17 @@
+from checks import AgentCheck
+try:
+    import psutil
+except ImportError:
+    psutil = None
+
+B2MB  = float(1048576)
+
+class SystemSwap(AgentCheck):
+
+    def check(self, instance):
+        if not psutil:
+            return {}
+
+        swap_mem = psutil.swap_memory()
+        self.gauge('system.swap.swapped_in', swap_mem.sin / B2MB)
```

Comment:
You should use a rate for these.
According to the psutil documentation, sin is 
```
the number of bytes the system has swapped in from disk (cumulative)
```

By submitting the metric as a rate, you will compute the # of bytes per second swapped in. 
Otherwise the graphs will just display a monotonically increasing value.

Also you should submit this value in bytes as per our discussion earlier.